specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -0,0 +1,1987 @@
+    },
+    "VirtualMachineSshCredentials": {
+      "type": "object",
+      "description": "Admin credentials for virtual machine",
+      "properties": {
+        "username": {
+          "description": "Username of admin account",
+          "type": "string"
+        },
+        "password": {
+          "description": "Password of admin account",
+          "type": "string"
+        },
+        "publicKeyData": {
+          "description": "Public key data",
+          "type": "string"
+        },
+        "privateKeyData": {
+          "description": "Private key data",
+          "type": "string"
+        }
+      }
+    },
+    "ComputeSecrets": {
+      "type": "object",
+      "description": "Secrets related to a Machine Learning compute. Might differ for every type of compute.",
+      "discriminator": "computeType",
+      "properties": {
+        "computeType": {
+          "description": "The type of compute",
+          "$ref": "#/definitions/ComputeType"
+        }
+      },
+      "required": [
+        "computeType"
+      ]
+    },
+    "AksComputeSecrets": {
+      "description": "Secrets related to a Machine Learning compute based on AKS.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ComputeSecrets"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "userKubeConfig": {
+              "type": "string",
+              "description": "Content of kubeconfig file that can be used to connect to the Kubernetes cluster."
+            },
+            "adminKubeConfig": {
+              "type": "string",
+              "description": "Content of kubeconfig file that can be used to connect to the Kubernetes cluster."
+            },
+            "imagePullSecretName": {
+              "type": "string",
+              "description": "Image registry pull secret."
+            }
+          }
+        }
+      ],
+      "x-ms-discriminator-value": "AKS"
+    },
+    "VirtualMachineSecrets": {
+      "description": "Secrets related to a Machine Learning compute based on AKS.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ComputeSecrets"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "administratorAccount": {
+              "description": "Admin creadentials for virtual machine.",
+              "$ref": "#/definitions/VirtualMachineSshCredentials"
+            }
+          }
+        }
+      ],
+      "x-ms-discriminator-value": "VirtualMachine"
+    },
+    "DatabricksComputeSecrets": {
+      "description": "Secrets related to a Machine Learning compute based on Databricks.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ComputeSecrets"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "databricksAccessToken": {
+              "description": "access token for databricks account.",
+              "type": "string"
+            }
+          }
+    
```

Comment:
Should this be `AMLCompute`? In many examples it is `AMLCompute` not `AmlCompute`. We need to make sure we are using the word `AMLCompute` in both examples and swagger spec, Please cross check all examples

Reply:
it should be AmlCompute ..